### DISCUSSION TITLE
OpenRadioss executables - Building machine Upgrade to RockyLinux 8

### DISCUSSION THREAD

**Discussion by MarianBulla** [OpenRadioss/OpenRadioss#2772] [Announcements]
Dear all, 
I would like to inform you, that due to technical reasons with CentOs 7 (beeind EndOfLife on June 30th), we are going to build our precpmiled executables for Linux with RockyLinux 8.

Who is affected?
Only users of OpenRadioss, who use the compiled executables on CentOs 7.

You still can compile it from source code on such machines.

Sorry for any inconvenience.

In case of any issues, please reach out to me.

Thanks and best regards,
Marian

**Comment by marcoserra68** [comment]
Hi Marian,
Thank you for the heads up. I am currently still running my cluster on CentOs 7 and plan to upgrade. I have a separate workstation that is running RockyLinux 9.  I was planning to upgrade my cluster to RockyLinux9 as well but now I am wondering if I should rather install RockyLinux8 instead to be perfectly aligned since OpenRadioss is going to be the principal solver I will use. Do you have any advice?
Best regards
Marco

**Reply by MarianBulla** [reply]
Hi Marco,

thanks for your interest in OpenRadioss and reaching out to me.
Yes, I think RockyLinux 9 should be fully ok.

Please, give it a try. And let us know.

With RockyLinux 9, you should be safe for a longer period. So go for it :-)

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by marcoserra68** [comment]
Hello Marian, thank you, yes it’s helpful.
Best regards
Marco

Marco Serra
Engenya GmbH
Oberhofen 118
8488 Turbenthal

Mobile: +41 79 317 20 88
Email: ***@***.***
Web: www.engenya.com
